lisp/ess-julia.el

```diff
@@ -46,6 +46,10 @@
 (require 'ess-utils)
 (require 'ess-r-mode)
 
+(defvar ac-prefix)
```

Comment:
Probably better to have `(require auto-complete)` inside of `ess-ac-julia-objects`.

Reply:
It is probably be a good thing to explicitly require packages inside the commands/functions that use them (ideally at entry points and not in all depended-on functions), but of course this won't actually help with the compilation warning. So these defvar/declare-function declarations are correct!

Reply:
Done